external/fv3viz/fv3viz/_plot_cube.py

```diff
@@ -543,3 +554,49 @@ def _plot_cube_axes(
     ax.set_global()
 
     return p_handle
+
+
+def _plot_scream_axes(
+    array: np.ndarray,
+    lat: np.ndarray,
+    lon: np.ndarray,
+    plotting_function: str,
+    ax: plt.axes = None,
+    **kwargs,
+):
+    if ax is None:
+        ax = plt.gca()
+    if plotting_function in ["tripcolor", "tricontour", "tricontourf"]:
```

Comment:
It'd say it's up to you if you want to maintain the different flavors of "tri" plotting; I'd say in practice we only ended up using one (pcolor).